sa/database.go

```diff
@@ -179,11 +178,9 @@ func ReportDbConnCount(dbMap *gorp.DbMap, statter metrics.Scope) {
 // https://godoc.org/github.com/coopernurse/gorp#DbMap.Insert
 func initTables(dbMap *gorp.DbMap) {
 	var regTable *gorp.TableMap
-	if features.Enabled(features.AllowAccountDeactivation) {
-		regTable = dbMap.AddTableWithName(regModelv2{}, "registrations").SetKeys(true, "ID")
-	} else {
-		regTable = dbMap.AddTableWithName(regModelv1{}, "registrations").SetKeys(true, "ID")
-	}
+
+	regTable = dbMap.AddTableWithName(regModelv2{}, "registrations").SetKeys(true, "ID")
```

Comment:
Can you remove the whitespace before/after this line? Thanks!

Reply:
In order to avoid formatting issues in future contributions, is there a guideline you guys follow?

Reply:
I think we've done a bad job of writing these sorts of things down.

@jsha @rolandshoemaker Can you think of a resource that would help here? Should we ticket writing one? I thought https://github.com/golang/go/wiki/CodeReviewComments might cover this but it doesn't seem to (and neither does Boulder's contrib markdown).

Reply:
Yep, it's worth writing down. I'm not sure what the rule would be; @cpu can you add your thoughts? In general my approach to blank lines is: use them like paragraph breaks to separate concepts; follow the local pattern (e.g. if there is a block of similar stuff, add to the block).